packages/framer-motion/src/value/types/numbers/index.ts

```diff
@@ -0,0 +1,17 @@
+import { clamp } from "../../../utils/clamp"
+
+export const number = {
+    test: (v: number) => typeof v === "number",
+    parse: parseFloat,
+    transform: (v: number) => v,
+}
+
+export const alpha = {
+    ...number,
+    transform: (v: number) => clamp(0, 1, v),
+}
+
+export const scale = {
+    ...number,
+    default: 1,
+}
```

Comment:
It seems to me these don't really belong a `numbers` utils

Reply:
The idea is they're flavours of `number` like `int` would be. It's sibling to unit types which contains `"px"` `"%"` etc

Reply:
👌🏻